Makefile

```diff
@@ -121,12 +121,12 @@ HTML_DST = \
 
 ## * install-rmd-deps : Install R packages dependencies to build the RMarkdown lesson
 install-rmd-deps:
-	Rscript -e 'source("bin/dependencies.R"); install_dependencies(identify_dependencies())'
+	Rscript -e 'source(file.path("bin", "dependencies.R")); install_dependencies(identify_dependencies())'
```

Comment:
`bin` and `dependencies.R` lose double quotations marks around them in Windows. Perhaps, we need to save this into a new `.R` file in the bin directory.